org-code-javabuilder/validation/src/main/java/org/code/validation/SystemOutTestRunner.java

```diff
@@ -0,0 +1,16 @@
+package org.code.validation;
+
+import java.util.List;
+import org.code.protocol.JavabuilderContext;
+import org.code.validation.support.ValidationProtocol;
+
+public class SystemOutTestRunner {
```

Comment:
Would it make sense to just have one `ValidationTestRunner` that runs the main method once, and has methods like `getSystemOutMessages`, `getNeighborhoodLog` etc? Do we anticipate validation writers needing to verify both system out and neighborhood messages in a single test?

Reply:
I wanted to make the api as simple as possible (so a single call both runs & gets the main method). We also shouldn't change the neighborhood api now since it's in use.